rest-service/manager_rest/storage/resource_models.py

```diff
@@ -1265,7 +1265,9 @@ class Event(SQLResourceBase):
             name='events__one_fk_not_null'
         ),
     )
-    id = db.Column(db.Text, index=True, default=lambda: str(uuid.uuid4()))
+    is_id_unique = False
+
+    id = None
```

Comment:
this is just to override the parent class attribute

Reply:
That should maybe be a comment in the code itself?

Reply:
I'll add that comment in next PR

Reply:
(added in this one)